python/meta.py

```diff
@@ -0,0 +1,3 @@
+NAME = ['DLRModel']
```

Comment:
This is a duplicate of `python/dlr/metadata.py`.

Reply:
removed.